### PR TITLE
Draft: Pending rename of package py-dockerpy-creds [py-wandb] added version 0.12.10

### PR DESCRIPTION
**Pull request by qwertos**

### CODE REVIEW

File: var/spack/repos/builtin/packages/py-wandb/package.py [hunk 1]
```diff
@@ -11,23 +11,33 @@ class PyWandb(PythonPackage):
...
+    depends_on('py-python-dateutil@2.6.1:', type=('build', 'run'))
+    depends_on('py-requests@2.0:2',         type=('build', 'run'))
+    depends_on('py-promise@2.0:2',          type=('build', 'run'))
+    depends_on('py-shortuuid@0.5:',         type=('build', 'run'))
+    depends_on('py-six@1.13:',              type=('build', 'run'))
+    depends_on('py-watchdog@0.8.3:',        type=('build', 'run'), when='@0.10.1')
+    depends_on('py-psutil@5.0:',            type=('build', 'run'))
+    depends_on('py-sentry-sdk@0.4:',        type=('build', 'run'))
+    depends_on('py-sentry-sdk@1:',          type=('build', 'run'), when='@0.12.10:')
+    depends_on('py-subprocess32@3.5.3:',    type=('build', 'run'))
```
Comment (adamjstewart) [inline var/spack/repos/builtin/packages/py-wandb/package.py]: This isn't required in the latest version

File: var/spack/repos/builtin/packages/py-wandb/package.py [hunk 2]
```diff
@@ -11,23 +11,33 @@ class PyWandb(PythonPackage):
...
+    depends_on('py-dockerpy-creds@0.4:',    type=('build', 'run'))
```
Comment (adamjstewart) [inline var/spack/repos/builtin/packages/py-wandb/package.py:37]:
```suggestion
    depends_on('py-docker-pycreds@0.4:',    type=('build', 'run'))
```
This package should be renamed to match PyPI.

File: var/spack/repos/builtin/packages/py-wandb/package.py [hunk 3]
```diff
@@ -11,23 +11,33 @@ class PyWandb(PythonPackage):
...
-    depends_on('py-shortuuid', type=('build', 'run'))
-    depends_on('py-six', type=('build', 'run'))
-    depends_on('py-watchdog', type=('build', 'run'))
-    depends_on('py-pyyaml', type=('build', 'run'))
-    depends_on('py-psutil', type=('build', 'run'))
-    depends_on('py-sentry-sdk', type=('build', 'run'))
-    depends_on('py-subprocess32', type=('build', 'run'))
-    depends_on('py-dockerpy-creds', type=('build', 'run'))
-    depends_on('py-configparser', type=('build', 'run'))
+    pypi     = "wandb/wandb-0.10.1.tar.gz"
```
Comment (adamjstewart) [inline var/spack/repos/builtin/packages/py-wandb/package.py]:
```suggestion
    pypi     = "wandb/wandb-0.12.10.tar.gz"
```

**Review by adamjstewart — CHANGES_REQUESTED** [review]

**Comment by adamjstewart** [comment]
Ping @qwertos

**Comment by alalazo** [comment]
Closing, since there has been no activity in a while. Feel free to reopen if you want to continue work on this.

**Pull request closed without merging**